ccw.js

```diff
@@ -67,22 +67,15 @@ const prompt = () => {
             },
         ])
         .then((answers) => {
-            exec(`npm i ${project}`);
-
-            let raw;
-
-            try {
-                raw = readFileSync("package.json");
-            } catch (err) {
-                throw err;
-            }
+            execSync(`npm i ${project}`);
 
             clone();
 
             if (answers["vite"]) {
                 console.log("Installing vite...");
 
-                exec("npm i vite", () => {
+                exec("npm i -D vite", () => {
+                    const raw = readFileSync("package.json").toString();
```

Comment:
I moved `readFileSync` to this place to make sure `raw` has the latest content of the `package.json` file after installing `vite`.